tools/jbrowse2/macros.xml

```diff
@@ -0,0 +1,468 @@
+<macros>
+    <token name="@TOOL_VERSION@">2.10.1</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">jbrowse2</requirement>
+            <requirement type="package" version="1.82">biopython</requirement>
+            <requirement type="package" version="0.7.1">bcbio-gff</requirement>
+            <requirement type="package" version="1.19">samtools</requirement>
+            <requirement type="package" version="6.0.1">pyyaml</requirement>
+            <requirement type="package" version="1.11">tabix</requirement>
+            <requirement type="package" version="4.6.0">findutils</requirement>
+            <requirement type="package" version="0.0.8">hictk</requirement>
+            <yield/>
+        </requirements>
+    </xml>
+    <token name="@DATA_DIR@">\$GALAXY_JBROWSE_SHARED_DIR</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <xml name="creators">
```

Comment:
I added this here, I hope its ok.

Reply:
of course - cleaner. Thanks for all this work @bgruening - much appreciated.